tmt/result.py

```diff
@@ -223,6 +222,18 @@ def show(self) -> str:
     def printable_note(self) -> str:
         return ', '.join(self.note)
 
+    def failures(self) -> list[Path]:
```

Comment:
Maybe `failure_logs`, as it does not return actual failures. And it can be a `@property`.